docker-entrypoint.sh

```diff
@@ -9,13 +10,19 @@ cleanup () {
 trap "cleanup" SIGINT
 trap "cleanup" SIGTERM
 
+cat-file-with-header() {
```

Comment:
Actually, this is not a valid bash function name (replaced to underscore after merge)